lib/ansible/plugins/action/include_vars.py

```diff
@@ -46,6 +48,7 @@ def _set_dir_defaults(self):
     def _set_args(self):
         """ Set instance variables based on the arguments that were passed """
 
+        self.hash_behaviour = self._task.args.get('hash_behaviour', None)
```

Comment:
Just simpler to use the constant here instead of None, would need no extra logic below